changelogs/fragments/1819-tidyup-pylint-blacklistnames.yml

```
@@ -0,0 +1,17 @@
+bugfixes:
+  - alternatives - replaced uses of ``_`` with ``dummy`` (https://github.com/ansible-collections/community.general/pull/1819).
```

How about something like:
```suggestion
  - "alternatives - internal refactoring: replaced uses of ``_`` with ``dummy`` (https://github.com/ansible-collections/community.general/pull/1819)."
```
Otherwise users might wonder what this means for them :)